src/components/toolbar/__tests__/__snapshots__/toolbar.test.js.snap

```diff
@@ -129,54 +129,7 @@ exports[`Toolbar Component should handle displaying secondary components, fields
 </Toolbar>
 `;
 
-exports[`Toolbar Component should handle updating toolbar chips: chips 1`] = `
```

Comment:
required: confirm this test removal

Reply:
the index for the chip needed updating. we went ahead and reverted the other indexes in test to retain the original snapshot